.github/workflows/lint.yml

```diff
@@ -4,11 +4,11 @@ on:
 jobs:
   golangci:
     name: lint
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: Run linter
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
```

Comment:
Upgraded to v3, because v2 doesn't work. On v2 I get 137 status code. It is only action version, version of linter stayed same